src/pages/Home.tsx

```diff
@@ -14,7 +14,7 @@ export const HomePage: React.FC = () => {
 
   return (
     <>
-      <Header onSearch={setSearchTerm} />
+      <Header onSearch={setSearchTerm} showSearch={true}/>
```

Comment:
Nit: Can refactor it like this.

```suggestion
      <Header onSearch={setSearchTerm} showSearch/>
```